src/app/spaces/space.service.ts

```diff
@@ -61,7 +61,7 @@ export class SpaceService {
           this.buildSpaceIndexMap();
           return space;
         })
-        .catch(this.handleError);
+        .catch((error) => {return this.handleError(error)});
```

Comment:
@joshuawilson Yes, I had issues referencing `this` in the catch handler, so had to invoke the handler instead of just passing a function reference to `.catch()`.